osu.Game.Tests/Beatmaps/IO/ImportBeatmapTest.cs

```diff
@@ -135,6 +135,53 @@ public async Task TestImportThenImport()
             }
         }
 
+        [Test]
+        public async Task TestImportThenExportThenImport()
+        {
+            using (HeadlessGameHost host = new CleanRunHeadlessGameHost(nameof(ImportBeatmapTest)))
+            {
+                try
+                {
+                    var osu = LoadOsuIntoHost(host);
+
+                    var temp = TestResources.GetTestBeatmapForImport();
+                    string extractedFolder = $"{temp}_extracted";
+                    Directory.CreateDirectory(extractedFolder);
+
+                    //Full path to the export folder created in Temp\of-test-headless
```

Comment:
Also I can't understand what is being tested here after three reads. Start by adding inline comments explaining what your intentions are